docs/en/Running-Kylin-on-Alluxio.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+title: Running Apache Kylin on Alluxio
+nickname: Apache Kylin
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Apache Kylin](http://kylin.apache.org/) on Alluxio, so
+that you can speed up the cube build phase in Kylin.
+
+# Prerequisites
+
+The prerequisite for this part is that you have
+[Java](Java-Setup.html). Alluxio cluster should also be
+set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
+[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Please [Download Kylin](http://kylin.apache.org/download/).
+
+# Configuration
+
+Kylin requires a setup environment including [Hive](http://hive.apache.org/),
```

Comment:
Yes, Hive is the data source, Hadoop is the build tool, Hbase is where the cube resides(like data sink).